wikidp/controllers/search.py

```diff
@@ -45,26 +49,28 @@ def get_search_result_context(search_string):
         for res in FileFormatExtSearchResult.search(search_string)
     ]
     # Check if searching with PUID
-    # pylint: disable=W1401
-    if re.search("[x-]?fmt/\d+", search_string) is not None:
+    if re.search(r"[x-]?fmt/\d+", search_string) is not None:
```

Comment:
regex here could probably be a named constant for clarity

Reply:
Good catch, just updated!